src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/LocalHelpers.cs

```diff
@@ -52,6 +52,18 @@ public static string GetGitDir(ILogger logger)
             return dir;
         }
 
+        public static string GetRootDir(ILogger logger)
+        {
+            string dir = ExecuteCommand("git", "rev-parse --show-toplevel", logger);
+
+            if (string.IsNullOrEmpty(dir))
+            {
+                throw new Exception("'.git' directory was not found. Check if git is installed and that a .git directory exists in the root of your repository.");
```

Comment:
>'.git' directory was not found. Check if git is installed and that a .git directory exists in the root of your repository. [](start = 37, length = 122)

We should probably make this match closer what the command does

Reply:
Improved this message a bit.

---
In reply to: [263629645](https://github.com/dotnet/arcade-services/pull/206#discussion_r263629645) [](ancestors = 263629645)

Reply:
Maybe something like "Root directory of the repo was not found. Check if git is installed and that your are in folder which contains a git repo (better wording might be needed..)."

---
In reply to: [264323058](https://github.com/dotnet/arcade-services/pull/206#discussion_r264323058) [](ancestors = 264323058,263629645)

Reply:
How's this?

---
In reply to: [264339694](https://github.com/dotnet/arcade-services/pull/206#discussion_r264339694) [](ancestors = 264339694,264323058,263629645)